testing/mailutils/APKBUILD

```diff
@@ -0,0 +1,74 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=mailutils
+pkgver=3.2
+pkgrel=0
+pkgdesc="GNU swiss army knife of electronic mail handling"
+url="https://mailutils.org/"
+arch="all"
+license="GPL3+"
+provides="mailx=$pkgver-r$pkgrel"
+replace="mailx"
+depends_dev="$pkgname-libs=$pkgver-r$pkgrel"
+makedepends="readline-dev libtool"
+checkdepends="autoconf"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-servers $pkgname-mh"
+source="https://ftp.gnu.org/gnu/mailutils/mailutils-$pkgver.tar.gz
+	disable-koi8-r-test.patch"
+builddir="$srcdir/mailutils-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr
+	make -j4
```

Comment:
Abuild exposes environment variable `JOBS` with user preferred number of jobs for `make` and similar. I’m not sure if `make` implicitly uses that variable and so whether it’s needed to be set explicitly. Anyway, if you want to set number of jobs, please use `-j $JOBS`.